test/ctia/stores/es/mapping_test.clj

```diff
@@ -4,229 +4,185 @@
              [document :as doc]
              [conn :as conn]
              [index :as index]]
-            [clj-momo.lib.es.document :refer [bulk-create-doc]]
+            [ctia.test-helpers.es :as es-helpers]
             [clojure.test :refer [deftest testing is]]))
 
 (deftest mapping-test
```

Comment:
Some context here, this is not trivial do not hesitate for any precision:
The `_all` field was removed in ES7. Instead, when no field is supplied for query_string queries in ES7, the query is applied on every fields and preserve existing mapping. This is a breaking change with our former use of `_all` field that included only some fields but analyzed them as text. This new behavior enable what was impossible: query every fields while respecting their own analyzers. Thus a keyword value is no more analyzed and a text when searching on every fields and will only be matched as if we filter only on it's value (which takes advantage of term queries: exact match ignoring case).